src/view/feed/feed.js

```diff
@@ -68,7 +69,7 @@ export const feed = () => {
             <br>
             
             
-            <label for="localização" class="label-class">Localização:</label>
```

Comment:
Isso foi sem querer, manter :  <label for="localização" class="label-class">Localização:</label>